### PR TITLE
Javascript file added to render contributors to contributors.html

### PR DESCRIPTION
**Pull request by Samcraig77**
Styling added for contributor cards as well.

### CODE REVIEW

File: assets/data/contributors.js [hunk 1]
```diff
@@ -1,37 +1,44 @@
-[
+export const contributorsArray = [
+  {
+    name: 'Sam Craig',
+    linkedIn: 'https://www.linkedin.com/in/sam-craig-059834252/',
+    gitHub: 'https://github.com/Samcraig77',
+    avatar: '/assets/images/contributors/Sam-avatar.jpg',
+    alt: 'Sam looking at the camera, with an out of focus tree in the background.'
```
Comment (michaella23) [inline assets/data/contributors.js:7]: Nice addition! I will add a comment on issue #26 to include alt description if sharing a picture.

File: html/contributors.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/index.css"> 
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.5.1/css/all.css">
+    <title>Contributors</title>
+</head>
+
+<body class="contributors-body">
+    <header>
```
Comment (michaella23) [inline html/contributors.html:12]: We'll want to include a path to the Header logo here as well, so the nav doesn't jump over to the left. Would be great if you could create a new issue for this. Let me know if you want to be assigned - should be a quick fix.

File: index.css [hunk 3]
```diff
@@ -284,22 +284,78 @@ background-color: red;
...
+}
+
+.contributor-card {
+  border: 2px solid var(--medium-gray);
+  border-radius: 8px;
+  text-align: center;
+  
+}
+
+.contributor-avatar {
```
Comment (michaella23) [inline index.css:313]: Did you play around with the height property on these? I notice we have a variety or image sizes being rendered. It's fine for now!

**Review by michaella23 — APPROVED** [review]
Thanks for all of the work you put in on this @Samcraig77! 

In the future, it might be better to have separate pull requests for the different kinds of changes - for example the changes to CONTRIBUTING don't really relate to this PR title. I think bugs will be easier to find if we organize our PRs as much as possible. I did see that you made lots of small commits with good commit messages throughout the whole process of making changes. That documentation is super helpful, so thank you.

File: assets/data/contributors.js (hunk 1 above)
Comment (Samcraig77) [reply assets/data/contributors.js:7]: Sorry, I went into the mountains (literally) after I submitted haha. Bad to submit then dip I know, sorry! Would love to work on any issues found 😊

File: html/contributors.html (hunk 2 above)
Comment (Samcraig77) [reply html/contributors.html:12]: Yes, please assign me! Or the person who did all the work on the header. Glad I changed my index.js haha.

File: index.css (hunk 3 above)
Comment (Samcraig77) [reply index.css:326]: I did with the .img-container, but will open an issue about this. Would love other people to look over add to/or fix some of my code.

**Pull request merged**